test/config_SUITE.erl

```diff
@@ -427,29 +433,29 @@ app(_Config)  ->
 	      ergw_aaa:add_config(services, K, Opts)
       end, Services),
 
-    RADIUS =
+    RADIUS = #{procedures =>
 	#{init => [#{service => <<"Default">>}],
 	  authenticate =>
 	      [maps:merge(?RADIUS_AUTH_CONFIG, ?RADIUS_SERVICE_OPTS),
 	       #{service => <<"Default">>, defaults => #{'TLS-Pre-Shared-Key' => <<"secret">>}}],
 	  authorize => [],
 	  start => [],
 	  interim => [],
-	  stop => []},
-    DIAMETER =
+	  stop => []}},
+    DIAMETER = #{procedures =>
 	#{init => [#{service => <<"Default">>}],
 	  authenticate => [?DIAMETER_SERVICE_OPTS],
 	  authorize => [],
```

Comment:
the content of `procedures` needs to be reindent.

Reply:
ok